tests/longevity-48h-verify-multiDC.yaml

```diff
@@ -1,47 +1,35 @@
-test_duration: 10080
-stress_cmd: cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(strategy=NetworkTopologyStrategy,us-east1-b=3,us-west1-b=3,us-east4-b=2)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..100000000000
-cassandra_stress_duration: 10080
-cassandra_stress_threads: 1000
-cassandra_stress_population_size: 100000000000
-#n_db_nodes: 3
-n_db_nodes: '5 5 3'
-n_loaders: 3
+test_duration: 2880
+prepare_write_cmd:  "cassandra-stress write cl=ALL n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"
+prepare_verify_cmd: "cassandra-stress read  cl=ALL n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=00200300..600200300"]
+stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=100m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
+n_db_nodes: '3 3'
+n_loaders: 1
 n_monitor_nodes: 1
-nemesis_class_name: 'MdcChaosMonkey'
-#nemesis_class_name: 'DrainerMonkey'
+nemesis_class_name: 'ChaosMonkey'
```

Comment:
Change to LimitedMonkey (check for exact name)

Reply:
changed to LimitedChaosMonkey